Magento_Bundle/templates/sales/order/items/renderer.phtml

```diff
@@ -104,7 +104,7 @@ $prevOptionId = '';
             <?php elseif ($item->getQtyShipped() > 0 && $item->getParentItem() && $block->isShipmentSeparately()) : ?>
                 <?= $block->escapeHtmlAttr(__('Shipped')) ?>:&nbsp;<?= /* @noEscape */ $item->getQtyShipped() * 1; ?>
             <?php else : ?>
-                <?= /* @noEscape */ $parentItem->getQtyOrdered() * 1; ?>
+                <?= $block->escapeHtmlAttr(__('Ordered')) ?>:&nbsp;<?= /* @noEscape */ $parentItem->getQtyOrdered() * 1; ?>
```

Comment:
shouldn't be here `escapeHtml`? it's not an attribute 🤔

Reply:
fixed

Reply:
thanks